source/globalCommands.py

```diff
@@ -66,6 +66,7 @@
 from base64 import b16encode
 import vision
 from utils.security import objectBelowLockScreenAndWindowsIsLocked
+from utils.synth import getOutputDeviceNames, setOutputDevice

```

Comment:
I'm slightly puzzled why this module is called synth while in fact, it contains logic for audio. I'm also not sure why this can't just reside in the nvwave module.

Reply:
>In fact, it doesn't contain logic for audio. Switching to another
audio device involves reinitializing tts engine and tones module.
I wanted to eliminate code duplication and decoupled this code.
If you have a better alternative i am glad to implement it.

Both of these functions are related to audio, not synthesizers.
I would rename the `utils.synth` module to `audio` or move to `nvwave` like @leonardder  suggests